src/rickbox/database/sql/schema/create_files.sql

```diff
@@ -1,14 +1,12 @@
-DROP TABLE IF EXISTS files;
```

Comment:
Backticks/quotes aren't required apparently, so I ripped them out to reduce visual noise.